### ISSUE TITLE
Hide gallery follower counts on GPPs

### ISSUE THREAD

**Issue by garrengotthardt** [artsy/force#161]
As we move follower counts into analytics emails we should hide them from GPPs (both 2.0 style GPPs and 1.0) for Subscribing Galleries.

We'll also be hiding institution and institutional seller counts but **not until early October** when a separate issue will follow

**Comment by garrengotthardt** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Hide Gallery Followers on GPPs (both 2.0 and 1.0 style pages) 🎣](https://trello.com/c/bON8Jrx0/578-hide-gallery-followers-on-gpps-both-2-0-and-1-0-style-pages)
